src/platform/site-wide/sass/modules/_m-medallia.scss

```diff
@@ -1,6 +1,17 @@
 /* Fonts */
 #liveForm .ng-binding,
-#liveForm div > span {
+#liveForm div > span,
+/* 
```

Comment:
I understand if y'all don't want this comment in the file. If that is the case, let me know, and I'll move it from the code to the PR description.